kqueen/blueprints/api/api3_0.yml

```diff
@@ -205,26 +205,180 @@ components:
+        provisioner:
+          type: "string"
+          description: "Reference to the related Provisioner"
           example: "Provisioner:<uuid>"
         state:
           $ref: "#/components/schemas/clusterState"
         kubeconfig:
           description: "Kubeconfig file in JSON format"
           type: "object"
         metadata:
+          description: "Azure-specific parameters, required for a cluster creation"
           type: "object"
+          properties:
+            location:
+              type: "string"
+              enum:
+                - "eastus"
+                - "centralus"
+                - "westeurope"
+            ssh_key:
+              type: "string"
+            node_count:
+              type: "integer"
+            vm_size:
+              type: "string"
+              enum:
+                - "Standard_D1_v2"
+                - "Standard_D2_v2"
+                - "Standard_D3_v2"
+                - "Standard_D4_v2"
+                - "Standard_D5_v2"
         created_at:
           type: "string"
           format: "date-time"
         owner:
           type: "string"
-          description: "Reference to related User"
+          description: "Reference to the related User"
+          example: "User:<uuid>"
+      required:
+        - "name"
+        - "provisioner"
+        - "owner"
+        - "metadata"
+    ManualCluster:
+      type: "object"
+      properties:
+        id:
+          type: "string"
+          format: "uuid"
+        name:
+          type: "string"
+        provisioner:
+          type: "string"
+          description: "Reference to the related Provisioner"
+          example: "Provisioner:<uuid>"
+        state:
+          $ref: "#/components/schemas/clusterState"
+        kubeconfig:
+          description: "Kubeconfig file in JSON format"
+          type: "object"
+        created_at:
+          type: "string"
+          format: "date-time"
+        owner:
+          type: "string"
+          description: "Reference to the related User"
+          example: "User:<uuid>"
+      required:
+        - "name"
+        - "provisioner"
+        - "owner"
+        - "kubeconfig"
+    JenkinsCluster:
+      type: "object"
+      properties:
+        id:
+          type: "string"
+          format: "uuid"
+        name:
+          type: "string"
+        provisioner:
+          type: "string"
+          description: "Reference to the related Provisioner"
+          example: "Provisioner:<uuid>"
+        state:
+          $ref: "#/components/schemas/clusterState"
+        kubeconfig:
+          description: "Kubeconfig file in JSON format"
+          type: "object"
+        metadata:
+          description: "Jenkins-specific parameters, required for a cluster creation"
+          type: "object"
+          properties:
+            override_parameters:
+              type: "object"
+              description: "key-value pairs, that overrides existent parameters for provided in config jenkins job"
```

Comment:
"key-value pairs, that override existent parameters for the ones provided in the jenkins job config"

Reply:
for provided job, not parameters. I'll ask Maria how to improve that

Reply:
"for the job provided in the jenkins config"?